frontend/src/metabase/common/components/DataPicker/hooks/useAvailableData.ts

```diff
@@ -8,6 +8,7 @@ interface Props {
 export const useAvailableData = ({ databaseId }: Props = {}) => {
   const { data, isLoading } = useSearchQuery(
     {
+      context: "available-models",
```

Comment:
We have multiple places where we use similar search API calls (with either `limit: 0` or `limit: 1`) just to get metadata.

Do we want to use `context` to specify **why** is the search API called, or **where** is it called?

If latter, then `useAvailableData` hook should receive `context` as a prop.

If former, then this should be called `"available-entities"` and we should update other places with similar Search API usage:
- https://github.com/metabase/metabase/blob/92d615073a3ba31710e26be9f2e83141352d1d1c/frontend/src/metabase/containers/NewItemMenu/NewItemMenu.tsx#L45-L48
- https://github.com/metabase/metabase/blob/f584ab658d6beae45a882dfc31abc9b4f01b7888/frontend/src/metabase/query_builder/components/DataSelector/DataSelector.jsx#L1111-L1126
- https://github.com/metabase/metabase/blob/0cec8d7d2bce995f08a250593d6dfae04d8cb55f/frontend/src/metabase/palette/hooks/useCommandPaletteBasicActions.tsx#L34-L37
- https://github.com/metabase/metabase/blob/20cfb4239902d51c4be219c83aa84f2893ceb5e3/frontend/src/metabase/search/components/filters/TypeFilter/TypeFilterContent.tsx#L20-L22
- https://github.com/metabase/metabase/blob/cbbb11be4ee476fe92b04733bbef1e0a402c7f7f/frontend/src/metabase/browse/metrics/BrowseMetrics.tsx#L159-L168
- https://github.com/metabase/metabase/blob/e98c0e38bc3883dab11d02f4a9a9b8e483be8def/frontend/src/metabase/browse/models/BrowseModels.tsx#L121-L130

Reply:
I did a search and made sure we're passing a context anywhere that we're using Search.loadList (and not passing a collection, since that will use CollectionItems). I ended up adding 2 new contexts for "query-builder" (native query builder has data reference panels) and "action-picker".